### DISCUSSION TITLE
Suggestion or Question.

### DISCUSSION THREAD

**Discussion by voipiran** [fonoster/goodtok#15] [Ideas]
Hi,
as you are Asterisk Specialist man,
Would it be possible to connect agent panel and Extensions to asterisk PBX?
for example the agents get calls from asterisk or they have asterisk extension?

thanks

**Comment by psanders** [comment]
Are you hoping to connect Goodtok to Asterisk?

**Comment by voipiran** [comment]
Yes that would be great,
goodtok works as a Asterisk Callcenter Agents Panel.

**Reply by psanders** [reply]
I see. It is possible that we may bring some of those features in the future.
